rc/tools/git.kak

```diff
@@ -109,21 +109,27 @@ define-command -params 1.. \
                       set-option buffer=$kak_bufname git_blame_flags '$kak_timestamp'
                   }" | kak -p ${kak_session}
                   git blame "$@" --incremental ${kak_buffile} | awk '
-                  function send_flags(text, flag, i) {
+                  function send_flags(flush,    text, i) {
```

Comment:
there is extra whitespace here

```suggestion
                  function send_flags(flush, text, i) {
```

LGTM otherwise, nice work!

BTW I haven't used the `git blame` wrapper much since I mostly use a binding that [launches `tig blame`](https://git.sr.ht/~krobelus/dotfiles/tree/f0fbf9c77bd413973b82899c73820034430e502b/item/.config/kak/kakrc#L433)

Reply:
> there is extra whitespace here

Yes, this is on purpose. From awk's man page:
```
Since functions were not originally part of the AWK language, the provision
for local variables is rather clumsy: They are declared as extra parameters
in the parameter list.  The convention is to separate local variables from
real parameters by extra spaces in the parameter list.  For example:

      function  f(p, q,     a, b)   # a and b are local
      {
           ...
      }

      /abc/     { ... ; f(1, 2) ; ... }

```

(Although I'm fine with removing the extra spaces if you prefer, it's not like I'd expect a lot of people to be familiar with that convention anyway.)

> BTW I haven't used the `git blame` wrapper much since I mostly use a binding that [launches `tig blame`](https://git.sr.ht/~krobelus/dotfiles/tree/f0fbf9c77bd413973b82899c73820034430e502b/item/.config/kak/kakrc#L433)

Never tried tig before, I've mostly worked with the fugitive plugin for Vim so far. I'll check it out, thank you for the pointer!

Reply:
> there is extra whitespace here

Yes, this is on purpose. From awk's man page:
```
Since functions were not originally part of the AWK language, the provision
for local variables is rather clumsy: They are declared as extra parameters
in the parameter list.  The convention is to separate local variables from
real parameters by extra spaces in the parameter list.  For example:

      function  f(p, q,     a, b)   # a and b are local
      {
           ...
      }

      /abc/     { ... ; f(1, 2) ; ... }

```

(Although I'm fine with removing the extra spaces if you prefer, it's not like I'd expect a lot of people to be familiar with that convention anyway.)

> BTW I haven't used the `git blame` wrapper much since I mostly use a binding that [launches `tig blame`](https://git.sr.ht/~krobelus/dotfiles/tree/f0fbf9c77bd413973b82899c73820034430e502b/item/.config/kak/kakrc#L433)

Never tried tig before, I've mostly worked with the fugitive plugin for Vim so far. I'll check it out, thank you for the pointer!